app/code/community/Bolt/Boltpay/controllers/ShippingController.php

```diff
@@ -221,7 +221,12 @@ public function prefetchEstimateAction()
             try {
                 /** @var Bolt_Boltpay_Helper_Api $helper */
                 $helper = Mage::helper('boltpay/api');
-                $estimateResponse = $helper->getShippingAndTaxEstimate($quote);
+
+                if (@$addressData['postcode']) {
+                    $estimateResponse = $helper->getShippingAndTaxEstimate($quote);
+                } else {
+                    $estimateResponse = $helper->getDefaultShippingAndTaxResponse();
```

Comment:
Let's just return here (no cache). I don't want to cache default response when `@$addressData['postcode']` is missing for some glitch